.spotless/licence.header

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2016 Google Inc.
```

Comment:
is this the desired version? 

There are currently three different version:

<img width="308" alt="image" src="https://github.com/user-attachments/assets/fe193958-55ca-404a-b7f6-14da3ef9a3d7" />